assets/js/modules/idea-hub/components/dashboard/DashboardIdeasWidget/index.stories.js

```diff
@@ -16,22 +16,125 @@
  * limitations under the License.
  */
 
+/**
+ * External dependencies
+ */
+import fetchMock from 'fetch-mock';
+
 /**
  * Internal dependencies
  */
+import API from 'googlesitekit-api';
 import DashboardIdeasWidget from './index';
 import { getWidgetComponentProps } from '../../../../../googlesitekit/widgets/util/';
 import { createTestRegistry, WithTestRegistry, provideModules } from '../../../../../../../tests/js/utils';
+import { enabledFeatures } from '../../../../../features';
+import * as fixtures from '../../../datastore/__fixtures__';
 
 const widgetComponentProps = getWidgetComponentProps( 'ideaHubIdeas' );
-export const Ready = () => <DashboardIdeasWidget { ...widgetComponentProps } />;
+const Template = ( { ...args } ) => <DashboardIdeasWidget { ...widgetComponentProps } { ...args } />;
```

Comment:
Let's also add stories for errors and loading variants of every tab.

Reply:
Error and Loading variants added, however I only added for the New tab since all the other tabs will have the same UI.